### PR TITLE
Support pip install in bigdl-nano

### PR DESCRIPTION
**Pull request by yangw1234**
- [x] support ipex

### CODE REVIEW

**Comment by yangw1234** [comment]
@leonardozcm could you help review this?

File: python/nano/setup.py [hunk 1]
```diff
@@ -21,40 +27,67 @@ def get_nano_packages():
...
-            ln = ln + " @ " + _dependency_links_tmp if _dependency_links_tmp else ln
-            _dependency_links_tmp = ""
-            _install_requires_list.append(ln)
-        else:
-            _dependency_links_tmp = ""
-    return _install_requires_list
+
+
+def download_libs():
+    url = "https://github.com/yangw1234/jemalloc/releases/download/v5.2.1-binary/libjemalloc.so"
```
[the remark this answers is not in the document]
Comment (yangw1234) [reply python/nano/setup.py:33]: I want to enable this even if user does not have conda installed.

File: python/nano/setup.py [hunk 2]
```diff
@@ -38,40 +44,67 @@ def get_nano_packages():
...
 
 def setup_package():
-    install_requires_list = load_requirements()
+
+    py_version = sys.version_info
+    ipex_version = "1.8.0"
+    ipex_links = parse_find_index_page("https://software.intel.com/ipex-whl-stable")
+    ipex_whl_name = f"torch_ipex-{ipex_version}-cp{py_version.major}{py_version.minor}" \
+                    f"-cp{py_version.major}{py_version.minor}m-linux_x86_64.whl"
+    ipex_link = ipex_links[ipex_whl_name]
```
Comment (leonardozcm) [inline python/nano/setup.py:99]: Skip if not found?

**Comment by leonardozcm** [comment]
I think you can make `install_requires` an optional package like bigdl-nano[pytorch], keep bigdl-nano the same as the old one.

**Pull request merged**